cocos/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@
 # build jscocos2d if BUILD_JS_LIBS=ON
 
 # The version number
-set(COCOS2D_X_VERSION 3.17)
+set(COCOS2D_X_VERSION 3.18)
```

Comment:
Do we know 3.18 release date?

Reply:
plan to do test in this month, release before Oct.31 if things goes well. or will finished in November.

> and next release may be named to 3.17.1 , for no big feature